### PR TITLE
From chars to words

### PR DESCRIPTION
**Pull request by Giansingh4710**
linted, has all words in util file, chars can make words

### CODE REVIEW

**Review by manjot0515 — APPROVED** [review]

**Review by manjot0515 — APPROVED** [review]

File: util/allWords.js [hunk 1]
```diff
@@ -0,0 +1,3727 @@
+export const words = [
```
Comment (tsingh777) [inline util/allWords.js:1]: can we put less hard coded words. just keep like 10 words while we build out the UI.

File: util/test.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import * as Anvaad from "anvaad-js";
```
Comment (tsingh777) [inline util/test.js:1]: remove file.

File: util/allWords.js (hunk 1 above)
Comment (Giansingh4710) [reply util/allWords.js:1]: Wouldn't it be better to have more words so that we have more of a chance of finding words that may cause problems.

Comment (tsingh777) [reply util/allWords.js:1]: We will be using the DB soon to query for words. we don't need to keep all the words in a file.

**Comment by tsingh777** [comment]
few lint errors. please fix

**Pull request closed without merging**